### DISCUSSION TITLE
rclone config for teldrive

### DISCUSSION THREAD

**Discussion by Jopo-JP** [tgdrive/teldrive#140] [Ideas]
Hi, i noticed that the sleep between api calls has been hardcoded at 400ms at teldrive.go at line 33-42
```go
const (
	minSleep         = 400 * time.Millisecond
	maxSleep         = 5 * time.Second
	decayConstant    = 2
	attackConstant   = 0
	timeFormat       = time.RFC3339
	maxChunkSize     = 2000 * fs.Mebi
	defaultChunkSize = 1000 * fs.Mebi
	minChunkSize     = 500 * fs.Mebi
)

``` 
It would be great if we could configure the sleep timer for teldrive so a remote with many folders can be scanned by rclone faster.
Otherwise we have to rebuild the entire rclone bin if we change the setting.

Thanks!

**Comment by divyam234** [comment]
@Jopo-JP  Its removed now  instantiated with default values check new build

**Comment by Jopo-JP** [comment]
> @Jopo-JP  Its removed now  instantiated with default values check new build

Hey. Yes it is now working perfect.

Thx
